requirements.txt

```diff
@@ -6,4 +6,4 @@ nlohmann/json@v3.9.1 -DJSON_MultipleHeaders=ON -DJSON_BuildTests=Off
 ROCmSoftwarePlatform/FunctionalPlus@v0.2.18-p0
 ROCmSoftwarePlatform/eigen@3.4.0
 ROCmSoftwarePlatform/frugally-deep@9683d557eb672ee2304f80f6682c51242d748a50
-ROCmSoftwarePlatform/composable_kernel@0629870d27397ab10a90fde6d7050f3e3d65fc2f -DDTYPES="fp16;fp32;bf16" -DCMAKE_BUILD_TYPE=Release -DINSTANCES_ONLY=ON
\ No newline at end of file
+ROCmSoftwarePlatform/composable_kernel@0c9a1d25b3951736bf1fdc480de5fa21208f8e56 -DDTYPES="fp16;fp32;bf16" -DCMAKE_BUILD_TYPE=Release -DINSTANCES_ONLY=ON
```

Comment:
@iq136boy it is fine now but in the future we should reference to CK's `mad-master` branch. ROCm release will be picked from that branch. Thanks!

Reply:
Since we have conflicts, let's see if we can apply the latest mainline of CK (amd-master) to this branch updates.

Reply:
> Since we have conflicts, let's see if we can apply the latest mainline of CK (amd-master) to this branch updates.

I think it should be fine. But let us see.